include/SFML/Graphics/Shader.hpp

```diff
@@ -58,8 +58,9 @@ class SFML_GRAPHICS_API Shader : GlResource, NonCopyable
     ////////////////////////////////////////////////////////////
     enum Type
     {
-        Vertex,  ///< Vertex shader
-        Fragment ///< Fragment (pixel) shader
+        Vertex =   1 << 0, ///< Vertex shader
+        Geometry = 1 << 1, ///< Geometry shader
+        Fragment = 1 << 2  ///< Fragment (pixel) shader
```

Comment:
Why did you make it so that these constants can be combined? I don't see where it would be used.